Assets/MixedRealityToolkit.SDK/Inspectors/Microsoft.MixedReality.Toolkit.SDK.Inspectors.asmdef

```diff
@@ -4,7 +4,8 @@
         "Microsoft.MixedReality.Toolkit",
         "Microsoft.MixedReality.Toolkit.Editor.Inspectors",
         "Microsoft.MixedReality.Toolkit.SDK",
-        "Microsoft.MixedReality.Toolkit.Services.InputSystem"
+        "Microsoft.MixedReality.Toolkit.Services.InputSystem",
+        "Microsoft.MixedReality.Toolkit.Tools"
```

Comment:
@keveleigh @davidkline-ms  I think that this places a dependency of the Foundation package onto the Tools package, which would create a circular dependency.

It's a bit unfortunate that this is something that we had to discover though through manual code review - ideally we'd have some tooling that would detect this (this last sentence is not directed at this PR, just a general comment that it's really hard to find these things in general without infra investments).

Reply:
Ah, I should clarify, the circular dependency isn't in the asmdefs (these are fine) - the circular-ness is at a higher level (i.e. the .unitypackage files), where now Tools.unitypackage depends on Foundation.unitypackage and Foundation.unitypackage depends on Tools.unitypackage.

Reply:
What is the specific interdependency that got added here? There might be room to move things around.

Reply:
The Inspector of ManipulationHandler  now has a button offering the user the possibility of migration to ObjectManipulator. MigrationTool was in Tools assembly. Moving MigrationTool to SDK.Experimental.Utilities and removing that dependency.   MigrationWindow stays on Tools, but Tools depending on SDK should be fine, right?

Reply:
Yep, Tool depending on SDK is totally fine. I *think* that doing what you said should be fine (usually circular reference removing involves doing something like this, or making some other root common thing that both can depend on. In this case because there's this root thing 'foundation' we're a little more limited).